packages/story-editor/src/app/uploader/useUploader.js

```diff
@@ -53,8 +54,14 @@ function useUploader() {
       ...allowedImageMimeTypes,
       ...allowedVectorMimeTypes,
       ...allowedVideoMimeTypes,
+      ...allowedAudioMimeTypes,
     ],
-    [allowedImageMimeTypes, allowedVectorMimeTypes, allowedVideoMimeTypes]
+    [
+      allowedImageMimeTypes,
+      allowedVectorMimeTypes,
+      allowedVideoMimeTypes,
+      allowedAudioMimeTypes,
```

Comment:
<img width="620" alt="Screenshot 2022-07-11 at 14 06 14" src="https://user-images.githubusercontent.com/237508/178271135-542e9995-155a-4b7e-b106-9db43de07488.png">

Allowing this, means that this message is now incorrect. It is not correctly possible to drap an audio file into canvas. See https://github.com/GoogleForCreators/web-stories-wp/issues/11206

Reply:
This also means unit tests are breaking.

Reply:
The issue is that if you add audio mime types to the allowed list, then it will appear on this message. It means you can drag an mp3 into canvas. A quick work around would be have some kind of override for this one case and fix in https://github.com/GoogleForCreators/web-stories-wp/issues/11206. Or just do work on https://github.com/GoogleForCreators/web-stories-wp/issues/11206